engine/table/src/main/java/io/deephaven/engine/table/impl/by/typed/TypedHasherFactory.java

```diff
@@ -626,8 +630,23 @@ public static String packageName(HasherConfig<?> hasherConfig) {
 
     @NotNull
     public static String hasherName(HasherConfig<?> hasherConfig, ChunkType[] chunkTypes) {
-        return hasherConfig.classPrefix
-                + Arrays.stream(chunkTypes).map(Objects::toString).collect(Collectors.joining(""));
+        if (chunkTypes.length <= 10) {
```

Comment:
Instead of "10" could we instead base our limit on the maximum filename lengths?